dspace-server-webapp/src/test/java/org/dspace/app/rest/TaskRestRepositoriesIT.java

```diff
@@ -1439,7 +1449,7 @@ public void approvalForbiddenTest() throws Exception {
                                            .withName("Sub Community")
                                            .build();
 
-        // the reviewer2 is a reviewer in a different step for the col1 and with the same role than reviewer1 for
+        // the reviewer2 is a reviewer in a different step for the colAA1 and with the same role than reviewer1 for
```

Comment:
Not sure why this comment needs changing? It looked correct, so I think this change can be reverted.

Reply:
From restoring from (for the indentations), re-implemented correction